app/templates/components/settings/application-section.hbs

```diff
@@ -1 +1,34 @@
-<p>Applications</p>
\ No newline at end of file
+<div class="ui row">
```

Comment:
@geekyd 
The outlet on `settings.hbs` is inside a `ui segment`, so I don't think `ui row` is required here. Correct me if I'm wrong :)